src/serialize/mod.rs

```diff
@@ -93,6 +93,7 @@ impl<'a> RawMsg<'a> {
             create::CREATE => Ok(mem::transmute(&self.bytes[0..(4 * 6)])),
             measure::MEASURE => Ok(mem::transmute(&self.bytes[0..8])),
             update_field::UPDATE_FIELD => Ok(mem::transmute(&self.bytes[0..4])),
+            ready::READY => Ok(mem::transmute(&self.bytes[0..(4 * 1)])),
```

Comment:
just `[0..4]` should be fine?

Reply:
it's just for clarification of where the number camera from (a single 4-byte int), just like how create has `4*6`